### PR TITLE
Add PythonOperator

### PR DESCRIPTION
**Pull request by jcraver1021**
Also includes
-BranchPythonOperator
-ShortCircuitOperator

These operators have the same arguments

### CODE REVIEW

**Comment by jcraver1021** [comment]
@peleyal for visibility

File: boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Etsy Inc.
```
Comment (peleyal) [inline boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml:1]: Should we move this out of this PR, as it is included in https://github.com/etsy/boundary-layer/pull/87?

Comment (dossett) [reply boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml:1]: Yes, please.  thanks!

**Review by dossett — CHANGES_REQUESTED** [review]
LGTM, I'll merge once the pubsub change is pulled out in favor of the other PR.

File: boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml (hunk 1 above)
Comment (jcraver1021) [reply boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml]: I reset to before that addition and reapplied my changes.

**Review by dossett — APPROVED** [review]

**Pull request merged**